visibility/test/test_sun_position/SunPositionAndHillshadeUnitTest.py

```diff
@@ -0,0 +1,127 @@
+
+import unittest
+import logging
+import arcpy
+from arcpy.sa import *
+import sys
+import traceback
+import datetime
+import os
+import TemplateConfigTest
+import TestUtilities
+                
+                
+class SunPositionAndHillshadeUnitTest(unittest.TestCase):
+    def setUp(self):
+        # set-up code
+        arcpy.AddMessage("Setting up SunPositionAndHillshadeUnitTest.")
+        self.checkPaths()
+        logging.basicConfig(filename='test.log',level=logging.DEBUG)
```

Comment:
@mfunk @JudTown17 - we'll probably want to use a "named logger"(so all messages get associated with that logger, in case muliple python loggers are in use) - the pattern is to first call `logger = logging.getLogger("SOME_COMMON_AGREEDUPON_NAME")` and just use that class variable in the TestCase class.  https://docs.python.org/2/library/logging.html#logging.getLogger 

(Note: that is why you were seeing those "Root" messages in your output - you weren't using a named logger